assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -484,9 +485,31 @@ export default Component.extend(TextareaTextManipulation, ComposerUploadUppy, {
 
   @discourseComputed("previewing")
   placeholder(previewing) {
-    return I18n.t(
-      previewing ? "chat.placeholder_previewing" : "chat.placeholder"
-    );
+    return previewing
+      ? I18n.t("chat.placeholder_previewing")
+      : this.messageRecipient(this.chatChannel);
```

Comment:
Now that you are passing chatChannel, you might want to init it to `null` at the top of the component.